app/controllers/glossary_terms/images_controller.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+# Clicking on an image currently fires a GET to these actions... because it
+# comes from a link made by thumbnail_helper#thumbnail(link: url_args)
+# with CRUD refactor, change thumbnail helper to fire a POST somehow?
+
+module GlossaryTerms
+  class ImagesController < ApplicationController
```

Comment:
Could this be a little more CRUDified by making it AttachmentsController, 
with actions
~~reuse~~ new
~~attach~~ create
~~remove~~ edit
~~detach~~ update
(This is one of the places I stumbled when I tried refactoring this.)

Reply:
Thanks for really considering this. (I initially did try it with just those CRUD _action_ names, but without your more clarifying _controller_ name `AttachmentsController`... I could just picture Jason looking over my shoulder and asking, "Are those names really clearer?" and thought, no indeed... `edit` is a misleading name in an `ImagesController` for a `detach` action. But it makes sense for an `AttachmentsController`.)

If you would like to commit those changes to all the attachment controllers in this branch, please do. Or, maybe better to do that in another PR. 

Don't consider anything in any of these controllers PRs set in stone, I'm thinking of the controller directory as one giant TODO list. I hope they will be in a constant state of digestion until the logic is in service objects and can be used for GraphQL directly.